supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystem.scala

```diff
@@ -198,8 +205,25 @@ trait SharedFileSystem extends PathFactory {
       PairOfFiles(src, dest)
     }
 
+    // A possibly staged version of the input file suitable for downstream processing, or just the original input
+    // file if no staging was required.
+    val staged: WomFile = value.mapFile { input =>
+      pathBuilders.collectFirst({ case http: HttpPathBuilder if HttpPathBuilder.accepts(input) => http }) match {
+        case Some(http) =>
```

Comment:
ToL for clarity, can `http` => `httpResolver`?

Reply:
sure